doc/source/getting_started/index.rst

```diff
@@ -3,64 +3,65 @@
 Getting started
 ===============
 
-.. grid:: 2
-    :gutter: 2 2 3 4
+PyMechanical is part of the broader `PyAnsys <pyansys_>`_ initiative,
+enabling the use of Ansys technologies directly from Python.
+This allows users to integrate the Mechanical multiphysics solver
+into custom applications.
+The ``ansys-mechanical-core`` package presents a Python-friendly
+interface to drive the software that facilitates the use of
+:ref:`ref_user_guide_scripting` commands.
 
-    .. grid-item-card:: :fas:`fa-regular fa-screwdriver-wrench` Installation guide
+
+.. grid:: 1 2 2 2
+
+    .. grid-item-card:: Background
+        :padding: 2 2 2 2
+        :link: ../architecture
+        :link-type: doc
+        :text-align: left
+
+        Information on the application architecture of Mechanical and why there are two Python interfaces.
+
+    .. grid-item-card:: Installation guide
+        :padding: 2 2 2 2
         :link: installation
         :link-type: doc
-        :text-align: center
+        :text-align: left
 
         How to install and verify PyMechanical.
```

Comment:
Does "verify" mean make sure PyMechanical is working correctly?

Reply:
Yes. check if mechanical is launching with pymechanical.